builtin/providers/google/resource_compute_global_address.go

```diff
@@ -14,7 +14,12 @@ func resourceComputeGlobalAddress() *schema.Resource {
 		Create: resourceComputeGlobalAddressCreate,
 		Read:   resourceComputeGlobalAddressRead,
 		Delete: resourceComputeGlobalAddressDelete,
-
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
```

Comment:
Same thing here: why not use [`schema.ImportStatePassthrough`](https://godoc.org/github.com/hashicorp/terraform/helper/schema#ImportStatePassthrough)?